rbac/server/api/users.py

```diff
@@ -17,34 +17,23 @@
 from uuid import uuid4
 import hashlib
 
-from environs import Env
-from sanic import Blueprint
 from sanic.response import json
+from sanic import Blueprint
+from environs import Env
 
 from rbac.common.crypto.keys import Key
 from rbac.common.crypto.secrets import encrypt_private_key, generate_api_key
 from rbac.common.user import User
+from rbac.common.user.delete_user import DeleteUser
 from rbac.server.api import utils
 from rbac.server.api.auth import authorized
 from rbac.server.api.errors import ApiBadRequest
-from rbac.server.api.proposals import compile_proposal_resource, PROPOSAL_TRANSACTION
+from rbac.server.api.proposals import compile_proposal_resource
 from rbac.server.db import auth_query
 from rbac.server.db import proposals_query
 from rbac.server.db import roles_query
 from rbac.server.db import users_query
-
 from rbac.common.logs import get_default_logger
-from rbac.common.sawtooth import batcher
```

Comment:
Something is not right with your rebase. Did you rebase against the latest in develop?

Reply:
addressed